src/main/java/ro/hiringsystem/model/InterviewerUser.java

```diff
@@ -0,0 +1,21 @@
+package ro.hiringsystem.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import ro.hiringsystem.model.abstracts.User;
+import ro.hiringsystem.model.enums.InterviewerType;
+
+import java.net.URL;
+import java.util.Optional;
+
+@SuperBuilder
+@Getter
+@Setter
+public class InterviewerUser extends User {
+
+    private InterviewerType interviewerType;
+
+    private String personalInformation;
```

Comment:
The 'personalInformation' name is not suggestive to the actual field. We already have personal information the User class, so if this it's intended to simulate the LinkedIn about me section, it should have a different name.